Eplant/tutorial.md

```diff
@@ -0,0 +1,227 @@
+  return (
+    <>
+      <Sidebar />
+      <EplantLayout />
+    </>
+  )
+}
+```
+
+
+## Eplant Layout
+Eplant Layout (`Eplant/EplantLayout.tsx`) hosts the actual views and ViewContainer (`Eplant/UI/Layout/ViewContainer/index.tsx`) is main component that does this. 
+
+`view` will be passed as a prop to the ViewContainer which will be rendered in 
+```
+<view.component
+    state={state}
+    geneticElement={gene}
+    activeData={activeData}
+    dispatch={dispatch}
+/>
+```
+
+
+### useViewData
+The `useViewDatai` function is a custom React hook designed to manage and load view data in the application. It leverages TypeScript generics to handle different types of view data, state, and actions.
+
+See `Eplant/View/viewData.ts` for more details.
+
+
+
+## View Interface
+Please see the `View` interface in `View \index.ts`
+This is the interface we use for most of the `views`
+
+The View interface is a TypeScript interface that defines the structure and functionality expected from a view component in a React application.
+
+
+### getInitialState:
+
+An optional method that returns the initial state of the view. If not defined, the state defaults to null.
+This method initializes the state when the view is first rendered.
+
+
+### getInitialData:
+
+A required method that fetches the data needed for the view based on a given gene and provides loading feedback.
+
+### reducer:
+
+An optional reducer function used to handle actions and update the state. This is useful for managing state changes based on different actions.
+
+
+### actions:
+
+An optional array of actions that can be shown in the view’s options menu. Each action allows interaction with the view.
+
+
+### component:
+
+A required React component that renders the view.
+Takes props of type ViewProps<Data, State, Action>.
+
+
+### icon:
+
+An optional React component that returns an icon representing the view. This is used in the gene info viewer to visually identify the view.
+
+### name:
+
+A required string that represents the name of the view. This is used to identify the view in various contexts.
+
+### id:
+
+A required string that provides a unique identifier for the view.
+
+
+### description:
+
+An optional string that provides a description of the view.
+
+### thumbnail:
+
+An optional string that represents the URL or path to a thumbnail image for the view.
+
+### citation:
+
+An optional React component that renders citations related to the gene. This helps in displaying references or additional information.
+Takes props which may include state, activeData, and gene.
+
+### header:
+
+An optional function that returns the title of the view's tab.
+Takes props of type `ViewProps<Data, State, Action>`.
+
+
+## state
+`state` (`Eplant/state/index.tsx`) is a React module that manages state using `Jotai` and provides various hooks and utility functions for managing application state in the application, which is similar to `Redux` and `React Hooks`.
```

Comment:
Can you include a link to [Jotai](https://jotai.org/docs/core/use-atom) here? Thanks